PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -232,9 +249,18 @@
                 }
             }
 
-            // manually masked views through view modifier `PostHogMaskViewModifier`
-            if view.phIsManuallyMasked {
-                maskableWidgets.append(view.toAbsoluteRect(parent))
+            // on RN, lots get converted to RCTRootContentView, RCTRootView, RCTView and sometimes its just the whole screen, we dont want to mask
+            // in such cases
+            if view.isNoCapture() || maskChildren {
+                let viewRect = view.toAbsoluteRect(parent)
+                let parentRect = parent.frame
+
+                // Check if the rectangles do not match
+                if !viewRect.equalTo(parentRect) {
+                    maskableWidgets.append(view.toAbsoluteRect(parent))
+                } else {
+                    maskChildren = true
```

Comment:
if the parent has the ph-no-capture label but its the whole screen, we still want to mask its children

Reply:
`parent.frame` is the UIWindow, maybe parent is a bad name but its the view of the whole screen